acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -64,6 +64,15 @@
   sitepp_content = generate_manifest(test_resources)
   assertion_code = generate_assertions(test_resources)
 
+  refutation_resources = test_resources.collect do |assertion_group|
```

Comment:
actually i'm not really sure why we need to isolate the `refute_match` entries.

thus was the "elegance" in the previous solution.  you could just run the assertions in each state (debug output, syslog, etc) and they would always and should always be the same.  i don't see why that wouldn't be the case here